src/components/SignUp/SignUp.css

```diff
@@ -63,7 +63,7 @@
   display: flex;
   flex-direction: row;
 }
-.sign-up-input {
+input {

```

Comment:
Can you add class selectors here? It might affect styling of other elements. Thank you!